docs/release-notes/1.5.2/release-1.5.2.md

```diff
@@ -48,6 +48,18 @@
 - Fixed OnnxTransformer output column mapping. ([#5192](https://github.com/dotnet/machinelearning/pull/5192))
 - Fixed version format of built packages ([#5197](https://github.com/dotnet/machinelearning/pull/5197))
 - Improvements to "Invalid TValue" error message ([#5189](https://github.com/dotnet/machinelearning/pull/5189))
+- Added IDisposable to OnnxTransformer and fixed memory leaks ([#5348](https://github.com/dotnet/machinelearning/pull/5348))
+- Fixes #4392 Add AddPredictionEnginePool overload for implementation factory ([#4393](https://github.com/dotnet/machinelearning/pull/4393))
+- Updated codegen to make it work with mlnet 1.5  ([#5173](https://github.com/dotnet/machinelearning/pull/5173))
+- Updated codegen to support object detection scenario. ([#5216](https://github.com/dotnet/machinelearning/pull/5216))
+- Fix issue #5350, check file lock before reload model ([#5351](https://github.com/dotnet/machinelearning/pull/5351))
```

Comment:
Recommend full URLs as otherwise the auto-numeric links break in github if a repo is renamed, or if viewing within a fork.
```suggestion
- Fix issue [#5350](https://github.com/dotnet/machinelearning/issues/5350), check file lock before reload model ([#5351](https://github.com/dotnet/machinelearning/pull/5351))
```